### PR TITLE
Enable to install prometheus chart as a dependency

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Ref https://github.com/pipe-cd/pipe/issues/1809

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by nakabonne** [comment]
@nghialv Removed downloaded prometheus chart. In case of running locally, they are required. So I'm going to look at how it goes the remote chart, once got merged.

File: manifests/pipecd/values.yaml [hunk 1]
```diff
@@ -107,3 +107,12 @@ cors:
...
 quickstart: false
+
+monitoring:
+  enabled: false
+
+# All directives inside this section will be directly sent to the prometheus chart.
+# Head to the below link to see all available values.
+# https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus/values.yaml
+# TODO: Prepare the default prometheus.yml
+prometheus: {}
```
Comment (nghialv) [inline manifests/pipecd/values.yaml:118]: How about moving this into the "monitoring" group?

Comment (nakabonne) [reply manifests/pipecd/values.yaml:118]: It is. Initialy I wanted to do so but couldn't. Tell me how to do it if you already know! If none I will give it a try to look into them.

Comment (nakabonne) [reply manifests/pipecd/values.yaml:118]: I little poked around how to do it, but none of them worked as well as we'd like.

Comment (nghialv) [reply manifests/pipecd/values.yaml:118]: 👍  It seems there is no way for that.

**Comment by khanhtc1202** [comment]
Way to go 🚀 
/approve

**Pull request merged**